windows/src/test/unit-tests/kmcomp/test_expansion.kmn

```diff
@@ -0,0 +1,15 @@
+c This file tests that expansions are valid, using kmdecomp to verify
+store(&NAME) 'Test Expansions'
```

Comment:
Is expansion a `store(&VERSION) "14.0"` feature?

Reply:
No, per the discussion, it's a compiler only feature that is backwardly compatible with v6.0.

Reply:
... with the v6.0 compiled format (hence, `store(&VERSION)`), but not the v6.0 compiler.

I suppose there's no flag / system store for required compiler version, huh?

Reply:
Yeah, we don't have any flag for minimum required compiler version at present. But most programming languages don't specify that, from what I can tell, so we are probably not doing anything too awful!